src/lib/actions/InitAction.ts

```diff
@@ -0,0 +1,9 @@
+
+
+export class InitAction {
+
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	public execute(input: object): void {
```

Comment:
This is fine for now. But we might want to add in a InitInput, a RunInput, RulesInput type so that the cli is responsible for parsing into these types.

Reply:
Yeah, that was my plan.